src/dss/utils.py

```diff
@@ -81,3 +113,79 @@ def get_lightkube_client(kubeconfig: Optional[str] = None):
     kubeconfig = KubeConfig.from_file(kubeconfig)
     lightkube_client = Client(config=kubeconfig)
     return lightkube_client
+
+
+def get_mlflow_tracking_uri() -> str:
+    return f"http://{MLFLOW_DEPLOYMENT_NAME}.{DSS_NAMESPACE}.svc.cluster.local:5000"
+
+
+def get_notebook_url(name: str, namespace: str, lightkube_client: Client) -> str:
+    """
+    Returns the URL of the notebook server given the name of the server.
+
+    Assumes the following conventions:
+    * the notebook server is exposed by a service of the same name
+    * the notebook service exposes the notebook on the first port in the service
+    """
+    service = lightkube_client.get(Service, namespace=DSS_NAMESPACE, name=name)
+    ip = service.spec.clusterIP
+    port = service.spec.ports[0].port
+    return f"http://{ip}:{port}/notebook/{namespace}/{name}/lab"
```

Comment:
This is incorrectl url the suffix /notebook/namespace/name/lab is not needed its not working with it, we need just the url.

Reply:
Syncing on this with Michal, this is true but we 'll have to remove the `NB_PREFIX` env variable from the notebook deployment yaml, given that it's not needed and it's a leftover from kubeflow.